src/test/resources/com/aws/greengrass/clientdevices/auth/ca_cert.pem.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
don't commit keys and certs. Even though these don't go to anything, appsec won't be happy. 

Generate the keys and certs during the test.